openwisp_monitoring/monitoring/configuration.py

```diff
@@ -468,6 +521,8 @@ def _get_access_tech():
 
 DEFAULT_CHARTS = {}
 
+DEFAULT_DASHBOARD_TRAFFIC_CHART = {'__all__': ['wan', 'eth1', 'eth0_2']}
```

Comment:
I think we should set `eth0.2` here and make the necessary changes so the system can handle this properly (it shall convert it automatically to `eth0_2` and we need a test for this.

Reply:
> I think we should set `eth0.2` here and make the necessary changes so the system can handle this properly (it shall convert it automatically to `eth0_2` and we need a test for this.

There's already a test interfaces added by user. I will update the code to also include the default interface in the conversion.